frontend/src/container/ListAlertRules/AlertsEmptyState/AlertsEmptyState.tsx

```diff
@@ -0,0 +1,233 @@
+/* eslint-disable jsx-a11y/click-events-have-key-events */
+/* eslint-disable jsx-a11y/no-static-element-interactions */
+/* eslint-disable @typescript-eslint/explicit-function-return-type */
+/* eslint-disable jsx-a11y/img-redundant-alt */
```

Comment:
Why are we disabling the function return type and img redundant alt ?

Reply:
fixed it